addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -230,13 +230,25 @@ ListRenderer.include({
                 }
             });
 
+             // Keep ref of the current row's widgets 
+             const tmpRefFieldWidget = self.allFieldWidgets[id];
```

Comment:
suggestion: currentRowFieldWidgets
(at least, make it plurial: there are several widgets)